lib/msal-common/src/cache/CacheManager.ts

```diff
@@ -884,6 +885,15 @@ export abstract class CacheManager implements ICacheManager {
                     return false;
                 }
             }
+
+            if (filter.tokenType === AuthenticationScheme.POP) {
```

Comment:
Maybe add comments here so we understand why we have this match logic in the future

Reply:
Added.